### ISSUE TITLE
Reorder tuning strips

### ISSUE THREAD

**Issue by j5v** [j5v/xetune#14]
As a user, I want to reorder the tuning strips.

Consider:
- Drag and drop
- Up/down buttons (good only for a few moves; tiresome for more)

Drag and drop is not accessible for keyboard-only users, so if drag and drop is done, it would be helpful to also provide keyboard-focusable button on each tuning strip that activates a move operation controlled with cursor keys, finalized by Enter/Space keys, and aborted by the Escape key.

**Comment by j5v** [comment]
New design option: a button on the tuning bar to promote it to the top as a reference tuning. This makes full reordering possible with a little thought, but the most foreseen use of this feature is to select a tuning as reference, or to group two items at the top.

**Comment by j5v** [comment]
Example of implementation. A **Promote as reference** icons displays for all tunings except the top tuning.
![image](https://github.com/j5v/xetune/assets/13245475/ea0653b5-92cd-42ef-917a-d16be8cfd899)

When you select **Promote as reference** on a tuning row, it moves to the top, and is used as reference. That means its lines drop through all the other tunings, for easy visual comparison.
![image](https://github.com/j5v/xetune/assets/13245475/ee563471-d1c0-4506-9582-173d1edcbfee)
